tests/programs/cyclic_imports/cyclic_importing_package/Child2.py

```diff
@@ -17,6 +17,8 @@
 #
 print("Child2:", __name__)
 
+# isort:start
```

Comment:
Excellent that you caught this and figured out how to solve that. It's not too well documented I have to admit.